packages/@uppy/transloadit/src/index.js

```diff
@@ -25,6 +25,10 @@ const sendErrorToConsole = originalErr => err => {
   console.error(error, originalErr)
 }
 
+const COMPANION = 'https://api2.transloadit.com/companion'
+// Regex matching acceptable postMessage() origins for authentication feedback from companion.
+const ALLOWED_COMPANION_PATTERN = /\.transloadit\.com$/
```

Comment:
Since this is a breaking change already, good opportunity to use more intention revealing names. `COMPANION` sounds like it contains the instance. 

```suggestion
const COMPANION_URL = 'https://api2.transloadit.com/companion'
// Regex matching acceptable postMessage() origins for authentication feedback from companion.
const COMPANION_URL_PATTERN = /\.transloadit\.com$/
```

Reply:
It's not a breaking change now, we decided to just add exports to the CDN bundle.